.github/workflows/pytest.yaml

```diff
@@ -8,32 +8,41 @@ on:
       - master
   release:
     types: [published]
-    
+
 jobs:
-  tests:
+  test:
```

Comment:
we don't have to rename it :) Otherwise we have to change the branch protection checks